src/Diag/Result.hs

```diff
@@ -195,7 +210,7 @@ renderSuccess :: [EmittedWarn] -> Doc AnsiStyle -> Maybe (Doc AnsiStyle)
 renderSuccess ws headerDoc =
   case notIgnoredErrs of
     [] -> Nothing
-    ws' -> Just $ header headerDoc <> subsection "Warning" (map renderEmittedWarn ws')
+    ws' -> Just $ header headerDoc <> unannotatedSubsection (map (renderEmittedWarn True) ws')
```

Comment:
Was debating if I should display the traceback for warnings on success. As is, the traceback is present on success, however I'm not sure how helpful it'll be bc users won't really care about those tracebacks if the code is functioning properly. So, it'll basically just consume screen space. Wanted to get thoughts on this.

Reply:
I think you asked the team, but I think your idea of keeping it in both the debug bundle and the file with the log and not having it present on stderr is a good one.